Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:latest
+
+WORKDIR /app
+COPY ./ /app
+RUN go mod download
+RUN go install github.com/codegangsta/gin@latest
+
+
+
```

Comment:
better to add `entrypoint.sh` and not to expose that to your container orchestrator (aka k8s, docker-compose,...)